//package pl.coderslab.exceptions;


//public class Main06 {

  //  public static void main(String[] args) {
     //   try {
     //   System.out.println(divide(5,0));
  //  }catch (IllegalArgumentException e){
 //           System.out.println( "Podano z≈Çy argument");
//        }

 //   static int divide (int a, int b)
//    {
    //    if(b==0){
   //         throw new IllegalArgumentException("b musi byc wieksze od 0");
   //     }
   //     int result = a / b;
   //     return result;
  //  }

//}
